Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "avail-light"
-version = "0.1.0"
+version = "2.0.0"
```

Comment:
The prev version was `0.1`. Avail version now is `1.1`. Should we bump this to `1.0` instead of `2.0`?

Reply:
@prabal-banerjee the old tag was never actually updated to 1.0 before. As we discussed, the amount of changes we introduced with the new architecture is such that this is more of a v2.0 than 1.1, so it might be better to bump it to 2.0 here.